internal/server/iigo.go

```diff
@@ -1,16 +1,65 @@
 package server
 
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"github.com/SOMAS2020/SOMAS2020/internal/server/iigointernal"
+)
+
 // runIIGO : IIGO decides rule changes, elections, sanctions
 func (s *SOMASServer) runIIGO() error {
 	s.logf("start runIIGO")
+	_ = iigointernal.RunIIGO(&s.gameState, &s.clientMap)
 	defer s.logf("finish runIIGO")
-	// TODO:- IIGO team
 	return nil
 }
 
 func (s *SOMASServer) runIIGOEndOfTurn() error {
 	s.logf("start runIIGOEndOfTurn")
 	defer s.logf("finish runIIGOEndOfTurn")
-	// TODO:- IIGO team
+	clientMap := s.clientMap
+	for i, v := range clientMap {
+		tax := v.GetTaxContribution()
```

Comment:
I wouldn't want to pay taxes if I were dead! :P  
Use `getNonDeadClients` :)

Reply:
I might call this func getAliveClients - glass half full not half empty :D

Reply:
okay to do this in helper.go?

Reply:
o nooooooo merge main ples i made it just now sorry :(